### DISCUSSION TITLE
.dumi/pages 路由的页面可以使用导航栏和侧边栏吗

### DISCUSSION THREAD

**Discussion by changhuaixin** [umijs/dumi#2117] [Q&A] [answered]
有没有样例参考，谢谢

**Comment by changhuaixin** [comment]
多问个问题，.dumi/pages 里路由的页面怎么支持中文导航

**Comment by PeachScript** [comment, marked as answer]
仓库有案例：https://github.com/umijs/dumi/blob/master/examples/normal/.dumi/pages/nothing.tsx

**Comment by PeachScript** [comment]
如果仍然遇到问题，需要提供最小可复现仓库及步骤，只提供描述无法排查
